android/festago/presentation/src/main/java/com/festago/festago/presentation/fcm/TicketEntryService.kt

```diff
@@ -9,7 +9,11 @@ import kotlinx.coroutines.runBlocking
 
 class TicketEntryService : FirebaseMessagingService() {
 
-    private val notificationManager by lazy { NotificationManager(this) }
+    private val notificationManager by lazy {
+        com.festago.festago.presentation.fcm.NotificationManager(
+            this
+        )
```

Comment:
한 줄에 작성이 안되는 걸까요??

Reply:
```suggestion
        NotificationManager(this)
```
파일 이동시 다음과 같이 변환되는 문제들이 있어서 모두 수정했다고 생각했는데 짜잘한게 남았네요 반영하겠습니다!